### ISSUE TITLE
[Firefox] Brief flash of white

### ISSUE THREAD

**Issue by kiwijam** [cheeaun/hackerweb#65]
When navigating to hackerweb, there's a brief but very noticable flash of white on Firefox desktop & mobile. Maybe background-color could be set earlier?

**Comment by cheeaun** [comment]
@kiwijam are you using dark mode?

**Comment by kiwijam** [comment]
Yep, dark mode on hackerweb (tried both 'auto' and 'on'), the browser and OS.

**Comment by cheeaun** [comment]
@kiwijam do you see this issue on other dark-mode sites?

**Comment by kiwijam** [comment]
Happens on hackerweb and discord. Doesn't happen with my other frequently visited dark mode websites. Tried with YouTube, Inoreader, reddit, Twitter, XDA, github and some forums I visit on mobile & desktop.

**Comment by Faqqy** [comment]
what if you try to use ajax?
